src/ClusterBootstrap/az_tools.py

```diff
@@ -129,7 +131,7 @@ def create_vm_pwd(vmname, vm_ip, vm_size, use_private_ip, pwd):
                     --nsg %s \
                     --admin-username %s \
                     --storage-sku %s \
-                    --data-disk-sizes-gb 2047 \
+                    --data-disk-sizes-gb 1023 \
```

Comment:
This is going to override Zhe's PR: https://github.com/microsoft/DLWorkspace/pull/503.
Please revise your change.

Reply:
Revert this one. To be fixed in another PR.